docs/aci-interation.md

```diff
@@ -0,0 +1,125 @@
+To use a volume when running a single container, specify storage account and fileshare name like this: 
+
+```
+docker run -v <storageaccount>/<fileshare>:/target/path[:ro]
+```
+  
+You can use one or more volumes, just use the `-v` option as many times as required.  
+In a compose application, you will defined volumes like this : 
+
+```yaml
+myservice:
+  image: nginx
+  volumes:
+    - mydata:/mount/testvolumes
+
+volumes:
+  mydata:
+    driver: azure_file
+    driver_opts:
+      share_name: myfileshare
+      storage_account_name: mystorageaccount
+```
+
+The volume short syntax in compose files cannot be used, as it is aimed at volume definition with local folders. Using the volume driver and driver option syntax in compose files makes the volume definition a lot more clear. 
+
+In both cases, when deploying the containers, the Docker cli will use your azure login to fetch the key to the storage account, and provide this key with the container deployment information, so that the container can access the volume.
+Volumes can be used from any fileshare in any storage account you have access to with your Azure login. You can specify `rw` or `ro` when mounting the volume (read/write by default).
+
+ACI Volumes can be managed with commands `docker volume ls`, `docker volume create` and `docker volume rm`. 
+Creating a volume requires a storage account name and a volume name. The storage account is created with default options if it does not exist, using ther region associated with your ACI context.  
+If the storage account name already exists (and you have access to it), a fileshare is created with default values, using this existing storage account. Volume creation fails if the fileshare also already exists.
+
+Listing volumes will list all fileshares you have access from your azure login, in any storage account. You can use existing fileshares as volumes, even if they have not been created throug the Docker CLI. 
+
+When using `docker volume rm`, the command will remove the specified fileshare from the specified storage account. If this storage account has no more fileshares, and if the storage account has been created by the DOcker CLI, then the storage account will also be deleted.   
+
+## Resource usage definition
+
+You can specify resources to be used by your containes, setting CPU and memory limits. 
+For single containers, use `docker run --cpu 2 --memory 2G`. 
+In compose files, you can use the service resource limits 
+
+```yaml
+services:
+  redis:
+    image: redis:alpine
+    deploy:
+      resources:
+        limits:
+          cpus: '0.50'
+          memory: 50M
+```
+
+## Environment variables
+
+Environment variables can be passed to ACI containers through the `docker run --env` flag.  
+Form compose applications, environment variables can be specified with the `--env-file` flag, or in the compose file with the `environment` service field. 
```

Comment:
I don't see the `--env-file` flag on `docker compose up`

```suggestion
For Compose applications, environment variables can be specified with the `--env-file` flag, or in the Compose file with the `environment` service field. 
```

Reply:
I confused this with env-file inside the compose file. doc fixed